.github/workflows/dev-image-update.yaml

```diff
@@ -54,44 +55,73 @@ jobs:
           ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
           ENABLE_SUBPROJECT_TASKS: true
       - name: "Build new delevop docker image"
-        uses: broadinstitute/datarepo-actions@0.36.0
+        uses: broadinstitute/datarepo-actions@sh-test
         with:
           actions_subcommand: 'gradlebuild'
           role_id: ${{ secrets.ROLE_ID }}
           secret_id: ${{ secrets.SECRET_ID }}
-      - name: "Check and edit Helm definition for dev"
-        uses: broadinstitute/datarepo-actions@0.36.0
+      - name: "Update Version in helm for Dev Env"
+        uses: broadinstitute/datarepo-actions@sh-test
         with:
           actions_subcommand: 'deploytagupdate'
           helm_env_prefix: dev
-      - name: 'Release Candidate Container Build: Get Previous tag'
-        id: apiprevioustag
-        uses: "broadinstitute/github-action-get-previous-tag@master"
-        env:
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
       - name: 'Release Candidate Container Build: Checkout tag for DataBiosphere/jade-data-repo'
         run: |
-          git checkout ${{ steps.apiprevioustag.outputs.tag }}
+          git checkout ${{ steps.bumperstep.outputs.tag }}
       - name: 'Release Candidate Container Build: Checkout DataBiosphere/jade-data-repo-ui repo'
         uses: actions/checkout@v2
         with:
           repository: 'DataBiosphere/jade-data-repo-ui'
           token: ${{ secrets.HELM_REPO_TOKEN }}
           path: jade-data-repo-ui
           ref: develop
```

Comment:
I actually don't know why we need this step. This image update is just for API, not UI.

Reply:
(I could try to test this out in a separate PR)